### ISSUE TITLE
`m` and `,` shortcuts not working

### ISSUE THREAD

**Issue by gsingh93** [BinaryAnalysisPlatform/qira#150]
The shortcuts `m` and `,` are not working for me on the latest version from Git. Pressing these keys leads to the following messages:

```
ERROR emit() got multiple values for keyword argument 'room' in navigatefunction with (0, 100, False)
ERROR emit() got multiple values for keyword argument 'room' in navigatefunction with (0, 100, True)
```

My system is Arch Linux.

**Comment by gsingh93** [comment]
It seems like a lot of the other keyboard shortcuts also don't do anything (space, i, d, c, a, etc.), but `m` and `,` are the only two where I see errors.

**Comment by nedwill** [comment]
I think this should be fixed by #158.
